app/templates/partials/flash_messages/archive_service_confirmation_message.html

```diff
@@ -0,0 +1,23 @@
+{% if platform_admin %}
+      <h3 class="govuk-body govuk-!-font-weight-bold">Before you delete ‘{{service_name}}’, confirm the user: </h3>
+           <ul class="govuk-list govuk-list--bullet">
+             <li>has given the exact name of the service they want to delete</li>
+             <li>is aware they will no longer be able to use the inbound number to receive replies if there is one</li>
+           </ul>
+
+      <h3 class="govuk-body govuk-!-font-weight-bold">You will need to:</h3>
+           <ul class="govuk-list govuk-list--bullet">
+              <li>check the user is a member of the service</li>
+              <li>check the user has the ‘Manage settings, team and usage’ permissions’</li>
+              <li>contact another team member if there is one with the ‘Manage setting,
+                  team and usage’ permissions for a second approval </li>
+              <li>check the service has not sent any messages over their allowance that we may need an
+                  to raise an invoice for </li>
+           </ul>
```

Comment:
```suggestion
      <h3 class="govuk-body govuk-!-font-weight-bold">You also need to:</h3>
           <ul class="govuk-list govuk-list--bullet">
              <li>check the user has the ‘Manage settings, team and usage’ permission for this service</li>
              <li>ask a team member with the ‘Manage settings, team and usage’ permission to confirm the request</li>
              <li>check if the service has exceeded its annual allowance, in case we need to raise an invoice</li>
           </ul>
```

The main change here is combining bullets 1 and 2 into a single instruction.

My other suggestions are minor edits to make the language shorter and more active.

Reply:
Thanks @karlchillmaid. Combining the first two bullet points makes sense.

Reply:
@BlessedDev Do we still need to commit this suggestion?

Reply:
I’ve made a new commit to update this.

Reply:
Thanks Karl. not sure how to accept suggestions for that?